src/menu/menu.scss

```diff
@@ -85,7 +89,7 @@
   
       > .iui-icon {
         @include themed {
-          fill: t(iui-icons-color-actionable-disabled);
+          fill: t(iui-icons-color);
```

Comment:
Why still different color when it was before?

Reply:
Someone must have screwed up 👀 It is fixed now.